.github/workflows/_test.yml

```diff
@@ -20,5 +16,6 @@ jobs:
           node-version: '${{ matrix.node-version }}'
       - name: Run unit tests
         run: |
+          [[ -f ./bin/ci-setup ]] && ./bin/ci-setup
```

Comment:
what does this do?

Reply:
Would it be worth hiding this behind an `npm run ci-setup || true` script so the file name and path may be varied in each repo?

Reply:
Yeah that's an option: I was weighing the possibilities when considering the implementation and actually this is the main things I'm looking to iterate on with feedback.

- First and foremost I'm looking to preserve consistency across Github Workflow files, having them identical across all our repos is a huge time saver when we update things.
- In the past, we've been pretty good about using `npm run` as the entrypoint for most scripts, so maintaining that consistency has some value for sure.
- On the flip side, I was actually thinking of avoiding the npm middleware layer, as it's caused problems in the past (see the signal swallowing in Docker that made us move away from `npm start` to `./bin/start`). Though there is also a [new builtin Node.js feature in v22](https://nodejs.org/en/blog/announcements/v22-release-announce#running-packagejson-scripts) that could help with that some day, or at least mean theres one less process running
- Being able to modify the path and filename, personally I don't see when we would need that, let me know if there's a particular case I'm forgetting

Reply:
Gonna merge this for now to unblock other repos but we can change it later if we come up with a better idea